### DISCUSSION TITLE
No such file or directory: 'subtitles.srt'

### DISCUSSION THREAD

**Discussion by cerohazuri** [ThioJoe/SRT-To-SSML#7] [General]
Hello, it does not read the srt file even though it is in the folder. thanks! 
![image](https://github.com/user-attachments/assets/d0585ef5-698e-4941-a38c-4f740837966f)

```
python C:\Users\Entel\Documents\GitHub\SRT-To-SSML\SRT-To-SSML.py
Traceback (most recent call last):
  File "C:\Users\Entel\Documents\GitHub\SRT-To-SSML\SRT-To-SSML.py", line 107, in <module>
    with open(srtFile, 'r', encoding='utf-8-sig') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'subtitles.srt'
```

**Comment by ThioJoe** [comment]
Does adding these two lines at the top of the script help? This should set the working directory to that of the script file.

```
import os
os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Also try just putting the name of the file instead of the entire path, or if you already were using just the file name, try putting the entire path.
